m1n1-raw.ld

```diff
@@ -77,6 +78,12 @@ SECTIONS {
         QUAD(0x544f424b43415453);
         PROVIDE(_stack_bot = .);
     } :data
+    .stack_el3 : ALIGN(0x4000) {
+        PROVIDE(_stack_top_el3 = .);
+        . += _stack_size_el3 - 8;
+        QUAD(0x544f424b43415453);
```

Comment:
This is a magic number to find the end of the binary. Please move this new stack before the regular stack and remove the magic.